content/intro-to-storybook/angular/en/using-addons.md

```diff
@@ -81,9 +81,9 @@ Now we can reproduce and work on this edge case with ease.
   />
 </video>
 
-If we are [visual testing](/intro-to-storybook/angular/en/test/), we'll also be informed if the ellipsizing solution breaks. Obscure edge-cases are liable to be forgotten without test coverage!
+If we are [visual testing](/intro-to-storybook/angular/en/test/), we'll also be informed if the ellipsizing solution breaks. Obscure edge cases are liable to be forgotten without test coverage!
```

Comment:
ellipsizing → truncating or truncation?

Reply:
truncating reads better. I've gone with it.